Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL := help
+SHELL = bash
+
+
+.PHONY: help
+help: ## show this help
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: install
+install: ## install dependencies
+	poetry install --with dev
+	poetry run pre-commit install
+
+.PHONY: lint
+lint: ## lint code
+	poetry run ruff check --fix coverage_comment tests
+	poetry run ruff format coverage_comment tests
```

Comment:
Pre-commit takes care of linting. A more appropriate (and more complete) lint command would be
```console
pre-commit run --all-files
```
Or even just
```console
pre-commit
```